ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_hyperbolic_functions.py

```diff
@@ -108,3 +108,54 @@ def test_numpy_tanh(
         subok=True,
         test_values=False,
     )
+
+
+# arcsinh
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=ivy_np.valid_float_dtypes),
+    dtype=st.sampled_from(ivy_np.valid_float_dtypes + (None,)),
+    where=np_frontend_helpers.where(),
+    as_variable=helpers.array_bools(num_arrays=1),
+    with_out=st.booleans(),
```

Comment:
the as_variable, with_out and native_array should not be given any value in the given section as they will be handled by the handle_cmd_line_args automatically.